classes/report/fields/runner/tests/uncompleted/cli.php

```diff
@@ -1,12 +1,12 @@
 <?php
 
-namespace mageekguy\atoum\report\fields\runner\tests\uncompleted;
+namespace mageekguy\atoum\report\fields\runner\tests\incompleted;
```

Comment:
the path of the file should also be changed

Reply:
oh yeah overlooked that one